Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -351,7 +353,27 @@ private void ShowOpenFileErrorDialog (Window parent, string filename, string pri
 			PintaCore.Chrome.ShowErrorDialog(parent, message, details);
 		}
 
-#region Public Events
+		private void ShowUnsupportedFormatDialog (Window parent, string filename, string primaryText, string details)
+		{
+			string markup = "<span weight=\"bold\" size=\"larger\">{0}</span>\n\n{1}";
+
+			string secondaryText = string.Format(Catalog.GetString("Could not open file: {0}"), filename);
+			secondaryText += $"{Environment.NewLine}{Environment.NewLine}Pinta supports the following file formats:{Environment.NewLine}";
```

Comment:
The "Pinta supports ..." string also needs to be translatable like the line above (i.e. using `Catalog.GetString()`).
https://github.com/PintaProject/Pinta/wiki/Translations has more details on how translations work

Reply:
Ok, I'll change it.